includes/autoloaders.inc.php

```diff
@@ -0,0 +1,73 @@
+<?php 
+
+/**
+ * @file includes/autoloaders.inc.php
+ *
+ * Copyright (c) 2014-2017 Simon Fraser University
+ * Copyright (c) 2000-2017 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @ingroup index
+ *
+ * @brief Registers custom autoloaders 
+ */
+
+// registering autoloader function for OJS namespace
+spl_autoload_register(function($class) {
+	$prefix = 'OJS\\';
```

Comment:
...but perhaps better than jamming everything into the `PKP` namespace would be to move the `spl_autoload_register` call for the application into the application repo.